benchmark/utils/TestsUtils.swift

```diff
@@ -80,7 +92,7 @@ public struct BenchmarkInfo {
   /// A set of category tags that describe this benchmark. This is used by the
   /// harness to allow for easy slicing of the set of benchmarks along tag
   /// boundaries, e.x.: run all string benchmarks or ref count benchmarks, etc.
-  public var tags: [BenchmarkCategory]
+  public var tags: Set<BenchmarkCategory>
```

Comment:
👍

Reply:
Instead of using a Set, please us an OptionSet.

Reply:
I'm not sure this can work, `BenchmarkCategory` is `String` backed enum (it is used for printing the categories) and IIRC the `OptionSet` requires enum's `rawValue` to be `FixedWidthInteger`. Am I missing something?

Reply:
Sure.